src/windows/FileHashProxy.js

```diff
@@ -0,0 +1,232 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+*/
+
+/* global Windows, WinJS, MSApp */
+
+var getFileFromPathAsync = Windows.Storage.StorageFile.getFileFromPathAsync;
+
+
+function cordovaPathToNative(path) {
+    // turn / into \\
+    var cleanPath = path.replace(/\//g, '\\');
+    // turn  \\ into \
+    cleanPath = cleanPath.replace(/\\+/g, '\\');
+    return cleanPath;
+}
+
+function nativePathToCordova (path) {
+    return path.replace(/\\/g, '/');
+}
+
+var driveRE = new RegExp("^[/]*([A-Z]:)");
```

Comment:
Can drives really start with a '/'?

Reply:
No clue, also from the file plugin.

Reply:
So what it's doing with this, is before it uses this, it's stripped off the `file://` portion of the path, leaving one slash. `file:///` URIs can include drive letters, so after that, it checks to see if there is a drive letter there, and replaces the `/C:` with `C:`.